src/core/internal/string.d

```diff
@@ -192,16 +192,22 @@ unittest
 
 int dstrcmp( scope const char[] s1, scope const char[] s2 ) @trusted
 {
-    import core.stdc.string : memcmp;
```

Comment:
This (dstrcmp change) really ought to be in its own commit... currently there is no rationale. I see it's to add CTFE support?